libsrc/pce/conio.s

```diff
@@ -0,0 +1,102 @@
+                .include "pce.inc"
+
+                .import vce_init
+                .import psg_init
+                .import vdc_init
+
+                .export initconio
+
+                .constructor initconio, 24
+
+                .macpack longbranch
+initconio:
+;;              jsr     vdc_init
+                jsr     vce_init
+                jsr     psg_init
+                jsr     conio_init
+                jsr     set_palette
+
+                st0     #VDC_RCR
+                st1     #<$0088
+                st2     #>$0088
+                rts
+
+                .import colors
+set_palette:
+                stz     VCE_ADDR_LO
+                stz     VCE_ADDR_HI
+
+                ldx #0
+@lp:
+                .repeat 16
+                lda colors,x
+                sta     VCE_DATA_LO
+                lda colors+1,x
+                sta     VCE_DATA_HI
+                .endrepeat
```

Comment:
That code is done only once.  It doesn't need to be fast.  So, it should be shortenned by rolling it into an inner loop.